src/Execution/Arguments/ArgumentSet.php

```diff
@@ -59,6 +60,12 @@ public function has(string $key): bool
         return isset($argument->value);
     }
 
+    /** Check if the ArgumentSet has a value with the given key. */
+    public function exists(string $key): bool
+    {
+        return Arr::exists($this->arguments, $key);
```

Comment:
Why not https://www.php.net/manual/en/function.array-key-exists.php?

Reply:
Makes sense. Updated!